### ISSUE TITLE
How to choose the best trained model

### ISSUE THREAD

**Issue by menghuaa** [ChenJoya/2dtan#12]
Hi, how do I select a model for testing when I have trained it? Does it rely on the loss or the test effect on the validation set? Your code does not give how to choose the best model.
